Sprint-2/debug/2.js

```diff
@@ -1,14 +1,22 @@
 // Predict and explain first...
 
-const num = 103;
+// const num = 103; // the variable num is in a global scope
```

Comment:
Yes, excellent and accurate. It is in the global scope. I was looking for this term specifically.